RELEASE_CHECKLIST.md

```diff
@@ -272,16 +273,15 @@ A release can be cut from any branch, but we assume you'll be doing it from `dev
     cargo publish -p apollo-router
     ```
 
-30. (Optional) To have a "social banner" for this release, run [this `htmlq` command](https://crates.io/crates/htmlq) (`cargo install htmlq`; its `jq` for HTML), open the link it produces, copy the image to your clipboard:
+30. (Optional) To have a "social banner" for this release, run [this `htmlq` command](https://crates.io/crates/htmlq) (`cargo install htmlq`, or on MacOS `brew install htmlq`; its `jq` for HTML), open the link it produces, copy the image to your clipboard:
 
     ```
     curl -s "https://github.com/apollographql/router/releases/tag/v${APOLLO_ROUTER_RELEASE_VERSION}" | htmlq 'meta[property="og:image"]' --attribute content
     ```
 
-### Review
+### prep PR Review
 
-Most review comments will be about the changelog. Once the PR is finalized and
-approved:
+Most review comments for the prep PR will be about the changelog. Once the prep PR is finalized and approved:
```

Comment:
fwiw, this section and even the cargo publishing stuff below is now obsoleted by the above steps.  based on largely 3/3 success in others trying these steps, i think next steps are actually turn these into a script.